content/blog/sivayanama-testing-opsramp-apis-with-pytest-fixtures.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Testing OpsRamp APIs with PyTest Fixtures
+date: 2023-07-26T06:58:31.750Z
+featuredBlog: true
+author: BalaSubramanian Vetrivel
+authorimage: /img/vetrivel-balasubramanian1-photo.jpg
+disable: false
+---
+<!--StartFragment-->
+
+# Testing OpsRamp APIs with PyTest Fixtures
+
+
+
+OpsRamp platform  provides rich set of APIs. By using these APIs customers can build soultion to automate various ITOM workflows such as discovery and monitoring , event and incident mangement and remidiation and automation. Testing these APIs, automated solution  are very critical to ensure reliability and stability of the same. PyTest is a powerful python testing framework and it is widely used to test APIs. Fixtures is one of the most important capabilities of PyTest framework. In this article, we will discuss some advanced techniques of testing OpsRamp APIs using PyTest fixtures.
+
+
+
+## What Fixtures are 
+
+PyTest fixtures are a special type of python function that provision fixed base line for testing.With the help of this base line we can ensure tests are run in reliable manner and produce consistent results and the same tests can be repeatable.
```

Comment:
Plase add a space character after ...for testing. With the help of ....
seond sentence: base line --> baseline